boot.php

```diff
@@ -38,9 +38,9 @@
 
 define ( 'FRIENDICA_PLATFORM',     'Friendica');
 define ( 'FRIENDICA_CODENAME',     'Asparagus');
-define ( 'FRIENDICA_VERSION',      '3.5.1-dev' );
+define ( 'FRIENDICA_VERSION',      '3.5.1-rc' );
```

Comment:
Same and more drastic here, if you later merge back on `develop`, you have to change this back again to **dev**.